src/platform/unittest/DataPathTest.cpp

```diff
@@ -427,7 +427,9 @@ QuicAddr DataPathTest::UnspecIPv4;
 QuicAddr DataPathTest::UnspecIPv6;
 
 struct CxPlatDataPath {
-    QUIC_EXECUTION_CONFIG DefaultExecutionConfig { QUIC_EXECUTION_CONFIG_FLAG_NONE, 10000, 0, {0} };
+    static uint8_t RawConfig[QUIC_EXECUTION_CONFIG_MIN_SIZE + 256 * sizeof(uint16_t)] {0};
+    static QUIC_EXECUTION_CONFIG* DefaultExecutionConfig;
```

Comment:
I don't understand why you made this change. The old stuff should work. You definitely don't need to fully specify the list of processors.

Reply:
`PollingIdleTimeoutUs == 10000` is for QTIP only?

Reply:
OK this value is the root cause